layers/+lang/prolog/packages.el

```diff
@@ -0,0 +1,74 @@
+;;; packages.el --- prolog layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
+;;
+;; Author: Newres Al Haider <newrescode@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst prolog-packages
+  '(
+    (prolog :location built-in)
```

Comment:
As mentioned above please add the packages the user can choose from here, and toggle with a config variable. This is how the other layers handle excluding package choices.